### PR TITLE
Convert oredict page to tags

### PR DESCRIPTION
**Pull request by williewillus**

### CODE REVIEW

File: docs/utilities/tags.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+Tags
+====
+
+Tags are generalized sets of objects in the game, used for grouping related things together and providing fast membership checks.
+
+Declaring Your Own Groupings
+----------------------------
+Tags are declared in your mod's datapack. For example, `/data/modid/tags/blocks/foo/tagname.json` will declare a `Tag<Block>` with ID `modid:foo/tagname`.
+Similarly, you may append to or override tags declared in other domains, such as vanilla, by declaring your own JSONs.
```
Comment (PaleoCrafter) [inline docs/utilities/tags.md]: vanilla -> Vanilla? I think it's capitalized almost everywhere, so better keep it consistent.
Also, I'd go with "writing your own JSONs" or something, you don't just declare the JSON file.

File: docs/utilities/tags.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+!!! note:
+    The `TagCollection` returned by `getCollection()` (and the `Tag`s within it) may expire if a reload happens, so you should always query the collection anew every time you need it.
+    The static `Tag` fields in `BlockTags` and `ItemTags` avoid this by introducing a wrapper that handles this expiring.
+
+
+Conventions
+-----------
+
+There are several conventions that will help facilitate compatibility in the ecosystem:
+* If there is a vanilla tag that fits your block or item, add it to that tag. See the [list of vanilla tags](https://minecraft.gamepedia.com/Tag#List_of_tags).
```
Comment (PaleoCrafter) [inline docs/utilities/tags.md]: The capitalization of `tag` isn't entirely consistent across the article, unless you wanted to distinguish between the abstract concept of "tag" and the code-linked "Tag", in which case I'd say the latter should also always be in inline code markup.

**Comment by williewillus** [comment]
addressed

File: docs/utilities/tags.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+Tags
+====
+
+Tags are generalized sets of objects in the game, used for grouping related things together and providing fast membership checks.
+
+Declaring Your Own Groupings
+----------------------------
+Tags are declared in your mod's datapack. For example, `/data/modid/tags/blocks/foo/tagname.json` will declare a `Tag<Block>` with ID `modid:foo/tagname`.
+Similarly, you may append to or override tags declared in other domains, such as Vanilla, by declaring your own JSONs.
+For example, to add your own mod's saplings to the Vanilla sapling tag, you would specify it in `/data/minecraft/tags/blocks/saplings.json`, and Vanilla will merge everything into one tag at reload.
```
Comment (tterrag1098) [inline docs/utilities/tags.md]: *If* replace = false.

File: docs/utilities/tags.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+As an example:
+```Java
+ResourceLocation myTagId = new ResourceLocation("mymod", "myitemgroup");
+Item unknownItem = stack.getItem();
+boolean isInGroup = ItemTags.getCollection().getOrCreateTag(myTagId).contains(unknownItem);
+// alternatively, can use getTag and perform a null check
+```
+
+!!! note:
+    The `TagCollection` returned by `getCollection()` (and the `Tag`s within it) may expire if a reload happens, so you should always query the collection anew every time you need it.
```
Comment (tterrag1098) [inline docs/utilities/tags.md]: Perhaps mention that resource reload listeners can be used to invalidate caches.

File: docs/utilities/tags.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+* For matching items in code, see the section above.
+* If you are declaring a new type of item grouping, follow a couple naming conventions:
+  * Use `domain:type/material`. When the name is a common one that all modders should adopt, use the `forge` domain.
+  * For example, brass ingots should be registered under the `forge:ingots/brass` tag, and cobalt nuggets under the `forge:nuggets/cobalt` tag.
+
+
+Using Tags in Recipes and Advancements
+--------------------------------------
+
+Tags are directly supported by Vanilla, see the respective Vanilla wiki pages for [recipes](https://minecraft.gamepedia.com/Recipe#JSON_format) and [advancements](https://minecraft.gamepedia.com/Advancements) for usage details.
```
Comment (tterrag1098) [inline docs/utilities/tags.md]: Use the index format for links

**Comment by tterrag1098** [comment]
I've FF'd 1.13.x to master, so there's some pointless conflicts in the oredictionary page now.

**Comment by williewillus** [comment]
addressed

**Pull request merged**